pkg/util/conditions/secret.go

```diff
@@ -40,3 +40,13 @@ func SetSecretOutdated(conditions *[]metav1.Condition, conditionType, message st
 	}
 	_ = meta.SetStatusCondition(conditions, condition)
 }
+
+func SetAuthSecretCreated(conditions *[]metav1.Condition, conditionType string, msg string) {
```

Comment:
how is this different from `SetSecretCreated`? 😅

Reply:
Moved to authtoken package because it is none-generic function to set specific condition message.